app/views/index/about.phtml

```diff
@@ -24,6 +24,24 @@
 	<h2><?= _t('index.about.bugs_reports') ?></h2>
 	<?= _t('index.about.github') ?>
 
+	<?php if (FreshRSS_Auth::hasAccess('admin')) { ?>
+		<details class="envInfo">
+			<summary><?= _t('index.about.bug_reports.environment_information') ?></summary>
+			<dl>
+				<dt><?= _t('index.about.bug_reports.environment_information.version_frss') ?></dt>
+				<dd><?= FRESHRSS_VERSION ?></dd>
+				<dt><?= _t('index.about.bug_reports.environment_information.version_php') ?></dt>
+				<dd><?= PHP_VERSION ?></dd>
+				<dt><?= _t('index.about.bug_reports.environment_information.database') ?></dt>
+				<dd><?= FreshRSS_Context::systemConf()->db['type'] ?></dd>
+				<dt><?= _t('index.about.bug_reports.environment_information.server_software') ?></dt>
+				<dd><?= $_SERVER['SERVER_SOFTWARE'] ?></dd>
```

Comment:
From PHPStan:
`Parameter #1 (mixed) of echo cannot be converted to string.`

I do not have any idea how to solve it correctly

Reply:
Maybe it'll like this?
```suggestion
				<dd><?= $_SERVER['SERVER_SOFTWARE'] ?? '' ?></dd>
```

Reply:
This would only help for `string|null` but not for mixed, which cannot always be casted to a string

Reply:
Why would it be mixed? I think phpstan just takes everything in $_SERVER as mixed which isn't accurate factually speaking afaik. Some are strings, some are arrays, but string values won't randomly be arrays.

Reply:
@Frenzie I agree. I believe PHPStan's reasoning is that upon arriving at this line, nothing prevents some code from before to have called something like `$_SERVER['SERVER_SOFTWARE'] = [5 => 'Bad idea', 2 => 3];`
But I would rather like preventing manual assignments to `$_SERVER` and keep the string guarantee. Any idea on how to do that would be welcome.

Reply:
* https://github.com/phpstan/phpstan/issues/8497
* https://github.com/phpstan/phpstan/issues/10870
* https://github.com/phpstan/phpstan/issues/12252

Reply:
If we can define an ignore rule that works well, that would be fine with me. Help welcome

Reply:
checks are green now. Thanks :)